packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/index.md

```diff
@@ -25,33 +25,26 @@ This use case requires the **password** and **phone** factors.
 
 ## Configuration updates
 
-Before you build the self-registration flow in your app, configure the Okta org to accept self-registration with both the password and phone factors by performing the following steps.
+Before you build a sign-in flow with password and phone factors, you need to configure the Okta org to accept both factors in your app:
 
-### Step 1: Set up your Okta org for password factor only use cases
+### Step 1: Set up your Okta org for a password factor
 
-If not already done, complete the steps described in [Set up your Okta org (for password factor only use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-password-factor-only-use-cases).
+Ensure your org is configured for the password factor by completing the steps in [Update the authenticators for password factor only use cases](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#update-the-authenticators-for-password-factor-only-use-cases).
 
-### Step 2: Set up your Okta org (for multi-factor use cases)
+### Step 2: Set up your Okta org for multifactor use cases
 
-If not already done, complete the steps described in [Set up your Okta org (for multi-factor use cases)](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases) section.
+Ensure your org is configured for multifactor use cases by completing the steps in [Set up your Okta org for multi-factor use cases](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-multifactor-use-cases).
```

Comment:
Another thought as well — this link and the previous one link to the .NET version of the instructions, not Java.

Reply:
Right ok, thanks for the explanation. Has this bug been reported?

Reply:
Robert reported this Jira for a bunch of treenav/stack-selector issues: https://oktainc.atlassian.net/browse/OKTA-413144